docs/inthewild.md

```diff
@@ -0,0 +1,31 @@
+# Archipelago Community
```

Comment:
We may ask @giancarlobi if he is OK with people looking at his Demo Byterfly instance or not. What do you think @alliomeria ? If so we can add a link her too!

Reply:
Will do this too, and will add little later in the week, along with additional updates. Does that sound ok @DiegoPino?